sidebars.ts

```diff
@@ -933,7 +945,150 @@ const sidebars: SidebarsConfig = {
+        description:
+          "Learn how to enable data-driven features and gradual rollouts.",
+      },
+      collapsed: true,
+      items: [
+        {
+          type: "html",
+          value: "New to FME?",
+          className: "horizontal-bar",
+        },
+        /* We could also do this way, if we want the tiles (for docs) to show on the Getting started page
+        {
+          type: "category",
+          label: "Getting started",
+          link: {
+            type: "generated-index",
+            slug: "feature-management-experimentation/getting-started",
+          },
+          collapsed: true,
+          items: [ //{ type: "autogenerated", dirName: "feature-management-experimentation", }
+            "feature-management-experimentation/getting-started/overview",
+            "feature-management-experimentation/getting-started/onboarding-guide",
+            "feature-management-experimentation/getting-started/tutorials/index",
+          ],
+        },
+        */
+        {
+          type: "autogenerated", 
+          dirName: "feature-management-experimentation/10-getting-started", 
+        },
+        {
+          type: "category",
+          label: "SDKs and infrastructure",
+          link: {
+            type: "generated-index",
+            slug: "feature-management-experimentation/sdks-and-infrastructure",
+          },
+          collapsed: true,
+          items: [ { type: "autogenerated", dirName: "feature-management-experimentation/20-sdks-and-infrastructure", }
+          ],
+        },
+        {
+          type: "html",
+          value: "Features",
+          className: "horizontal-bar",
+        },
+        {
+          type: "category",
+          label: "Feature management",
+          link: {
+            type: "generated-index",
+            slug: "feature-management-experimentation/feature-management",
+          },
+          collapsed: true,
+          items: [ { type: "autogenerated", dirName: "feature-management-experimentation/40-feature-management", }
+          ],
+        },
+        {
+          type: "html",
+          value: "Management & Admin",
+          className: "horizontal-bar",
+        },
+        {
+          type: "category",
+          label: "Integrations",
+          link: {
+            type: "generated-index",
+            slug: "feature-management-experimentation/integrations",
+          },
+          collapsed: true,
+          items: [ { type: "autogenerated", dirName: "feature-management-experimentation/80-integrations", }
+          ],
+        },
+        "feature-management-experimentation/fme-support",
+      ],
+    },
+    /* {
+      type: "category",
+      label: "FME Feature releases",
+      collapsed: false,
+      collapsible: false,
+      items: [ */
+        // Release Notes
+        {
+          type: "link",
+          label: "Release Notes",
+          className: "sidebar-Release_Notes",
+          href: "https://www.split.io/releases/",
```

Comment:
Small URL update that I noticed in the preview link!

```suggestion
          href: "/release-notes/feature-management-experimentation/",
```

Reply:
Thank you, nice catch!